contents/blog/uh-i-like-my-job.md

```diff
@@ -0,0 +1,126 @@
+---
+date: 2024-12-12
+title: Uh, I like my job?
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - danilo-campos
+category: Culture, People
+---
+
+I never thought it would happen to me.
+
+I was an ambitious kid, and I always paid close attention to successful people talking about their careers on TV. A lot of folks seemed to love their jobs.
+
+I wanted that someday.
+
+But two decades into my career, nothing had really clicked. I liked *aspects* of some jobs. I liked certain kinds of work.
+
+Still, over and over again, familiar dysfunctions showed up in the workplace. Sometimes I would complain, pitch changes. Sometimes these were adopted, but most times I was dismissed as an impractical idealist.
+
+Or worse: a malcontent.
+
+So I gave up on ever liking a job. Eventually I gave up on *jobs* altogether, hoping to scrape a living from a pile of grit and contracts.
+
+Then I found PostHog. Two months in, I’m still finding the experience to be *extremely weird.* It seems as though everything that made previous experiences of work into bullshit can be solved through **hygiene**.
+
+## Honesty from leadership
+
+It’s an all-remote workplace, with an aggressive focus on writing things down. But that’s basic, everyone knows that advice.
+
+What feels distinct is that PostHog’s small executive team doesn’t exempt itself from this approach.
+
+My experience of the typical job is that executive decision making is a rat’s nest of hand-waving, loose reasoning, and ass-covering.
+
+But my first week on the job, I saw our co-founders writing up some mundane, quarters-long failure of their assumptions, extracting the lesson that could be learned from it.
+
+This sat in a GitHub issue, in an internal repo anyone could read.
+
+Every quarter we all get to read the same income statements that investors do. This is only proper: we are all of us investors ourselves, after a fashion, deferring cash today for equity we hope to increase in value tomorrow.
+
+I got into startups 14 years ago, but I’ve never gotten this degree of transparency at work. Being transparent keeps leaders honest.
+
+## Rigor enables autonomy
+
+I’ve been here two months and no one has made me look at a fucking slideshow.
+
+Slideshows are the refuge of the corporate bullshitter. A means of eliding detail through headlines and text that never quite coheres into an *argument*, and never finds itself as a useful reference in the future.
```

Comment:
I think I'd drop "bullshitter" here, too. You've already used "ass covering", so don't reuse it instead.

Reply:
How about "snow job artist"